src/services/TeamHub/TeamHub.WebApi/controllers/UserController.cs

```diff
@@ -0,0 +1,31 @@
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using TeamHub.BLL.MediatR.CQRS.Users.Queries;
+
+namespace TeamHub.WebApi.controllers;
+
+[ApiController]
+[Authorize]
+[Route("api/[controller]")]
+public class UserController : ControllerBase
+{
+    private readonly IMediator _mediator;
+
+    public UserController(IMediator mediator)
+    {
+        _mediator = mediator;
+    }
+
+    /// <summary>
+    /// Get Users
+    /// </summary>
+    [HttpGet]
+    public async Task<IActionResult> GetComment(CancellationToken cancellationToken)
```

Comment:
For token you can use HttpContext.RequestAborted. In case of this you do not need to pass token in controller method. Only in handler